mappings/net/minecraft/entity/ai/memory/SensorTypes.mapping

```diff
@@ -0,0 +1 @@
+CLASS aoo net/minecraft/entity/ai/memory/SensorTypes
```

Comment:
`Sensor`, not `SensorTypes`. But I already mapped this as well as its subclasses and fields, so it can be removed.